tests/ftest.sh

```diff
@@ -22,7 +22,7 @@ MAX_DURATION=600
 export REFRESH_RATE="${REFRESH_RATE:-5}"
 export DATA_SIZE="${DATA_SIZE:-medium}"
 export RUNNING_FTEST=True
-export VERSION='8.9.0-SNAPSHOT'
+export VERSION='8.10.0-SNAPSHOT'
```

Comment:
Not sure about this but I think this will make the nightly fail until there's an 8.10 snapshot available? cc @tarekziade

Reply:
Yup this should be bumped in a different PR and merged later